lib/runner.rb

```diff
@@ -52,11 +69,10 @@ def messages_per_channel
       end
   end
 
-  def message_payloads
+  def message_payloads(messages_per_channel, msg_for_one, msg_for_other)
```

Comment:
the parameter `messages_per_channel` shadows the method here, so we should either rename it to something different or rename the method.

Reply:
Also thinking about the methods in this class, it seems strange that you have to pass all the messages around as a single structure.

I'm assuming that each channel is independent of the others, so you could change the methods to operate on a single channel instead of mapping and looping inside the method bodies.

We could also extract a class (`Channel` or something) to hold the messages for a channel, and move the responsibility for generating the method payload to that.